addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -115,7 +115,7 @@ def _onchange_project_id(self):
     @api.depends('employee_id')
     def _compute_user_id(self):
         for line in self:
-            line.user_id = line.employee_id.user_id if line.employee_id else line._default_user()
+            line.user_id = line.employee_id.user_id if line.employee_id else line.env.context.get('user_id', line.env.user.id)
```

Comment:
Isn't it what self._default_user() returns? Why changing it?

Reply:
Method doesn't exist anymore

Reply:
Ok, but why then? We try to avoid business logic repetition. I see this pattern twice in the diff. Why having removed it?